libsolidity/formal/CHC.cpp

```diff
@@ -1487,10 +1500,12 @@ optional<string> CHC::generateCounterexample(CHCSolverInterface::CexGraph const&
 			{
 				auto inValues = summaryPredicate->summaryPostInputValues(summaryArgs);
 				auto const& inParams = calledFun->parameters();
-				localState += formatVariableModel(inParams, inValues, "\n") + "\n";
+				if (auto inStr = formatVariableModel(inParams, inValues, "\n"); !inStr.empty())
+					localState += inStr + "\n";
 				auto outValues = summaryPredicate->summaryPostOutputValues(summaryArgs);
 				auto const& outParams = calledFun->returnParameters();
-				localState += formatVariableModel(outParams, outValues, "\n") + "\n";
+				if (auto outStr = formatVariableModel(outParams, outValues, "\n"); !outStr.empty())
+					localState += outStr + "\n";
```

Comment:
These `if`s are responsible for the 350 or so examples changed where only newlines were removed, right? :)

Reply:
Yep, there were too many useless blank lines